### ISSUE TITLE
Services dont work with library v2.x 

### ISSUE THREAD

**Issue by kool79** [googleapis/google-api-java-client#2265]
Every service (actually tested on Drive nd Sheets)  has version check in static initializer:
```
 static {
    com.google.api.client.util.Preconditions.checkState(
        com.google.api.client.googleapis.GoogleUtils.MAJOR_VERSION == 1 &&
        com.google.api.client.googleapis.GoogleUtils.MINOR_VERSION >= 15,
        "You are currently running with version %s of google-api-client. " +
        "You need at least version 1.15 of google-api-client to run version " +
        "1.25.0 of the Google Sheets API library.", com.google.api.client.googleapis.GoogleUtils.VERSION);
  }
```
Bacause of this I have an error when use the latest google-api-java-client v2.2.0 with the latests versions of services (1.25.0): 
You are currently running with version 2.2.0 of google-api-client. You need at least version 1.15 of google-api-client

**Comment by mpeddada1** [comment]
@kool79 Thanks for filing this issue! The error you are seeing is looking very similar to https://github.com/googleapis/google-api-java-client/issues/2137. Could you help us confirm if you see the same issue when using `google-api-client` v1.35.2?

**Comment by kool79** [comment]
Hi, @mpeddada1 
I did not try to use 1.35.2 but I think it will work fine with client v1.25

Actually I fixed my issue by updating to the latest version of servise (I use google-api-services-drive and google-api-services-sheets). The reason was my wrong assumption that I use the latest version of client.

Initially I use **mvnrepository.com** to search for the latest versions  of services. But unfortunately this repository **does not contain the latest version**. It shows that the latest version is 1.25, which does not work with google-api-java-client v2.x.x:
![scr_2023_02_23T18_50_58](https://user-images.githubusercontent.com/4625317/220976376-2cb0cfe0-4e95-409e-a946-a338f3056ea4.png)

But then I use https://search.maven.org/ and got the actual letest version for may services:
![scr_2023_02_23T18_58_46](https://user-images.githubusercontent.com/4625317/220976929-499dffc5-3fca-454a-b7f8-85f6695e0e89.png)
